tools/who_what_benchmark/whowhatbench/model_loaders.py

```diff
@@ -504,6 +504,7 @@ def load_embedding_genai_pipeline(model_dir, device="CPU", ov_config=None, **kwa
     config.max_length = EMBED_DEFAULT_MAX_LENGTH
     config.normalize = kwargs.get("embeds_normalize", False)
     config.pad_to_max_length = True
```

Comment:
@sbalandi I think we need to set it to `false` by defalt and add `pad_to_max_length` argument to wwb.
Most of the cases It needed for NPU device to fix model shapes. For CPU device if input prompt is shorter than `EMBED_DEFAULT_MAX_LENGTH` we just introduce unnesesary additional computations.
For the momemnt `EMBED_DEFAULT_MAX_LENGTH=200` do you know average test dataset prompt lengths?

Reply:
No, I was thinking about switching `pad_to_max_length` to `False` by default. Considering avg is 307 for this dataset, let's skip it for now